### DISCUSSION TITLE
Table caption?

### DISCUSSION THREAD

**Discussion by katycroffbell** [knowledgefutures/pubpub#2076] [General]
How do I add a caption to a table?

**Comment by gabestein** [comment]
Hi Katy — thanks for writing in. Right now, we don't have native support for table captions, although it's something we'd like to add as part of a larger table overhaul. Folks typically either use a super or subscript below a table to simulate a caption, or, more commonly, create a cell at the bottom of the table that spans all the columns and use that for captions.

**Reply by katycroffbell** [reply]
Thanks, Gabe. Good to know so I don't waste more time trying to add a caption. And FWIW, I support adding that capability!

**Reply by gabestein** [reply]
👍. Discussed a bit more [here](https://github.com/pubpub/pubpub/issues/2084). We'll get to it!

**Reply by katycroffbell** [reply]
Oh boy, an enhancement request page? Buckle up.

**Reply by katycroffbell** [reply]
Hi again, I tried adding a row at the top and merging cells to create space for a table caption, but for some reason I can't highlight multiple cells to merge them. I tried in both Chrome and Firefox. I've done it before and don't think anything has changed on my end.

**Comment by katycroffbell** [comment]
Another table question! 

When exporting a pub as a pdf, it forces table columns to be all the same width. Is there any way around this? I'm guessing no, but thought I'd ask anyway.

<img width="433" alt="image" src="https://user-images.githubusercontent.com/92737253/180289024-e0e6c7ea-4745-4539-bd59-08f8b955aa2e.png">
